security/pkg/stsservice/tokenmanager/tokenmanager.go

```diff
@@ -41,14 +42,33 @@ type Config struct {
 	TrustDomain string
 }
 
-func getGcpProjectNumber() string {
+// GcpProjectInfo stores GCP project information, including project number,
+// project ID, cluster location, cluster name
+type GcpProjectInfo struct {
```

Comment:
should be `GCPProjectInfo` . This does not seem to follow golang naming best practice, by changing the casing in the middle of an acronym)

Reply:
All of the naming fixes are in https://github.com/istio/istio/pull/20823.